src/core/server/config/test_utils.ts

```diff
@@ -16,18 +16,22 @@ function collectDeprecations(
 ) {
   const deprecations = provider(configDeprecationFactory);
   const deprecationMessages: string[] = [];
+  const deprecationLevels: string[] = [];
```

Comment:
Needed to test changing the `rewriteCorsSetting` level from critical to warning.